### PR TITLE
Node 16 Support

### PR DESCRIPTION
**Pull request by Newbie012**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,6 @@
     "@types/adm-zip": "0.4.32",
     "@types/cli-progress": "1.8.1",
     "@types/node": "12.7.2",
```
Comment (nirdshabo1988) [inline package.json:39]: shouldnt we update this one as well?

Comment (Newbie012) [reply package.json]: done

**Comment by nirdshabo1988** [comment]
not working.
didnt handled .parseMessage. what causing a hang on the server project.

**Comment by Newbie012** [comment]
closing in favor of https://github.com/MedFlyt/mfsqlchecker/pull/12

**Pull request closed without merging**